packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -336,6 +336,8 @@ class ResolutionRequest {
         file = potentialModulePath + '.native.js';
```

Comment:
I haven't followed the mjs spec closely but I assume that we are going to implement whatever resolution is done between .js and .mjs (if it's not already working today)